packages/loot-core/src/server/accounts/sync.ts

```diff
@@ -575,6 +580,7 @@ export async function reconcileExternalTransactions(acctId, transactions) {
 
       // Update the transaction
       const updates = {
+        date: trans.date,
```

Comment:
Also to note: this should be the only functional change. Everything else should be exactly the same as before.

Reply:
This would revert #1559, and personally I think it's a bad idea since it removes agency from the user to correct the dates.  (I for example semi-regularly change the dates after shopping at places that batch-submit transactions multiple days after they occur (sometimes over a week >.>), because to me it matters more when _I_ made the transaction than when my bank actually booked it.)

Reply:
That's an excellent point!

I've made this operation conditional: https://github.com/actualbudget/actual/pull/2534/commits/0edcd096d18c436e29d95ad2bb19131f317d743f